### PR TITLE
spice/deb_version - added the spice on top of master

### PR DESCRIPTION
**Pull request by iambibhas**
Reviving old stale branch `ideas/deb`. Added on top of master, as mentioned in #1667. Should close that other PR.

### CODE REVIEW

**Comment by jagtalon** [comment]
@iambibhas looks good to me. Take it away @abeyang https://ddh9.duckduckgo.com/?q=debian+versions+for+emacs24&ia=software

**Comment by abeyang** [comment]
@iambibhas we're trying to phase out the zebra stripes -- can we make it look more like the **whois IA**:

https://duckduckgo.com/?q=whois+facebook.com&ia=whois

**Comment by moollaza** [comment]
@iambibhas @abeyang we actually added default styling now for the record template. If you turn off `rowHighlight` and remove your CSS it should look the same as Whois :)

**Comment by iambibhas** [comment]
Here's the new look -

**Comment by abeyang** [comment]
@iambibhas looking better! Just need to sprinkle in some css love:

```css
.record__cell--key {
  /* separate the columns a bit more */
  padding-right: 2.5em;
}
```

**Comment by abeyang** [comment]
Sweet. Looks good! :+1:

File: lib/DDG/Spice/DebVersion.pm [hunk 1]
```diff
@@ -0,0 +1,26 @@
+# Get the version of a package in various debian repositories
```
Comment (moollaza) [inline lib/DDG/Spice/DebVersion.pm]: Mind putting this below the package declaration? The canonical approach is to set that as the `#ABSTRACT` or better yet the `description` for the metadata.

File: lib/DDG/Spice/DebVersion.pm [hunk 2]
```diff
@@ -0,0 +1,26 @@
...
+package DDG::Spice::DebVersion;
+
+use Text::Trim;
+use DDG::Spice;
+
+attribution github => ['https://github.com/iambibhas', 'Bibhas'],
```
Comment (moollaza) [inline lib/DDG/Spice/DebVersion.pm]: Please fill out the missing metadata: https://duck.co/duckduckhack/metadata

**Comment by iambibhas** [comment]
@moollaza why is the IA url  https://duck.co/ia/view/dev_version? Shouldn't it be deb_version?

**Comment by moollaza** [comment]
@iambibhas sigh, yes it should. I'll make a new page...

File: lib/DDG/Spice/DebVersion.pm (hunk 2 above)
Comment (moollaza) [inline lib/DDG/Spice/DebVersion.pm]: Just noticed this wasn't finished yet! I was looking for a sample query now that it's deployed...

**Comment by moollaza** [comment]
@iambibhas congrats! It's live: debian versions for emacs24

Thanks again for taking the time to contribute! We really appreciate it.

**Pull request merged**